lib/routes/account.js

```diff
@@ -1172,12 +1170,12 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
           }
         }
       },
-      handler: function accountDestroy(request, reply) {
+      handler: async function accountDestroy(request, h) {
```

Comment:
What is `h`? I don't see it being used...

Reply:
Hapi invented this https://github.com/hapijs/hapi/blob/master/API.md#toolkit-properties

Reply:
Okay, but if we're not using it, why are we adding it to the signature? Isn't eslint just going to complain about unused variables?

Reply:
@philbooth they added it everywhere https://hapijs.com/tutorials/getting-started?lang=en_US#user-content-adding-routes but I'd be happy to not use it

Reply:
Yep I get that, my point is if we're not actually using it why are we going to the trouble of adding it to the signature on our side.

By analogy, every time I use `Array.forEach` I don't do this:

```js
[ 1, 2, 3 ].forEach((item, index, array) => console.log(item))
```

I do this:

```js
[ 1, 2, 3 ].forEach(item => console.log(item))
```

Reply:
But I'm bike-shedding, I realise. It doesn't actually matter. It just seemed like a weird effort to go to, adding an esoteric and confusingly named unused parameter to every single route handler.

Reply:
Yeah we should remove it yeah